### DISCUSSION TITLE
Self-hosted NextCloud connection issue after updating Floccus to v5.0.10. This is ERROR occurs "E017: Network error..."

### DISCUSSION THREAD

**Discussion by mvozzo** [floccusaddon/floccus#1573] [General]
My Brave browser updated my Floccus extension from v4.19.0 to v5.0.10.
My Version: Nextcloud Hub 6 (27.1.7) 
I now get the following error when connecting to my self-hosted Nextcloud server hosted at local address https://192.168.0.111 
 
`E017: Network error: Check your network connection and your account details Last synchronized: 0 seconds ago`

There is no connection issues as I have logged into my Nextcloud server in the same browser. 

I have done the following and they have all failed:
1) Under "Floccus Options" I click "Login with Nextcloud" and I am prompted to accept the risks. Did not fix the issue. 
2) I Removed my Profile and created a new Profile
3) In the same browser, I log out of NextCloud and then logged back in with "Accept the Risk" prompt checked.
4) I even restarted my NextCloud Server and did all of the above

Any recommendations?  
If I can't get this working, how can I revert back to the old v4.19.0 and not have Brave auto-update Floccus?

Thanks.

**Comment by marcelklehr** [comment]
I assume you are using a self-signed certificate? That is apparently not supported in Chrome's manifest v3 anymore. I have no way to help you with that except recommend getting a proper certificate from e.g. letsencrypt
